src/lib/ai/providers.ts

```
@@ -1,30 +1,39 @@
+import { createGoogleGenerativeAI } from '@ai-sdk/google';
+import { openai } from '@ai-sdk/openai';
+import { env } from '@/env';
+
+const hackclub = createOpenAICompatible({
+  name: 'hackclub',
+  apiKey: env.HACKCLUB_API_KEY,
+  baseURL: 'https://ai.hackclub.com',
+});
+
+const openrouter = createOpenRouter({
+  apiKey: env.OPENROUTER_API_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)